README.md

```diff
@@ -75,11 +74,20 @@ Basically, **Buyers can redeem their mTokens for governance tokens once the lock
 
 Liquidation can be triggered before the end of the locking period only by unanimous votes.
 
-
 ## Setup
 
 ```
 forge build
 
 forge test
 ```
+
+**Note**: to test the oracle, you will need a **Ethereum Mainnet RPC** node address to run the forked network tests. Such as from [Infura](http://infura.io/).
+
+Once you have a mainnet RPC node url, set it to the `MAINNET_RPC_URL` variable in the `.env.example` file and rename the file to `.env`.
+
+If you would like to **omit the oracle tests** then instead of `forge test` run
+
+```bash
+forge test --no-match-contract Oracle  # or --nmc for short
+```
```

Comment:
We should also add some details about deployment, as this is getting more complex
```suggestion
## Deploy

First deploy the OracleConsulter library:

`forge create src/libraries/oracle/OracleConsulter.sol:OracleConsulter`

Then use its address to deploy the main contract:

`forge create src/MoltenFunding.sol:MoltenFunding --libraries src/libraries/oracle/OracleConsulter.sol:OracleConsulter:<OracleConsulter contract address> ...`
```

Reply:
As the oracle logic in this codebase are all libraries, they are not to be deployed.

Reply:
...which is why functions which just perform calculations (`view` and `pure`) should be considered to be grouped into libs to reduce deployed contract code size.